ansible/vars.yml

```diff
@@ -24,8 +24,8 @@ postgrest_release: "12.2.3"
 postgrest_arm_release_checksum: sha1:fbfd6613d711ce1afa25c42d5df8f1b017f396f9
 postgrest_x86_release_checksum: sha1:61c513f91a8931be4062587b9d4a18b42acf5c05
 
-gotrue_release: 2.175.0
-gotrue_release_checksum: sha1:4e2a41e08936195daf443c60201691c1bd991709
+gotrue_release: 2.176.0
```

Comment:
this needs to be 2.176.1 and the shasum needs to be updated to `3b392d2e332d5b5114e571c0d62ab9f827fd5b76`